### PR TITLE
Implementation with imported widgets

### PR DESCRIPTION
**Pull request by LeonardoMolleker**
ezgif com-gif-maker (12)

Implementation of the movie app with the widgets of the movie_widgets repository.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -24,6 +24,9 @@ dependencies:
   flutter:
     sdk: flutter
   http: ^0.13.3
+  exported_movie_widgets:
+    git:
+      url: https://github.com/LeonardoMolleker/movie_widgets
```
Comment (MarcosCGlobant) [inline pubspec.yaml]:
```suggestion
  exported_movie_widgets:
    git:
      url: https://github.com/LeonardoMolleker/movie_widgets
      ref: main
```

you should include the reference to the branch inside of the repository, as the above example

Comment (LeonardoMolleker) [reply pubspec.yaml]: Done

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**